components/src/core/components/CardTable/Table/TableHeaderSortDropdown.vue

```diff
@@ -71,14 +93,26 @@ export default defineComponent({
   },
 
   methods: {
-    openDropdown() {
+    openDropdown($e :Event) {
       this.isActive = true;
+      this.$nextTick(() => {
+        const firstFocusableElement = ($e?.target as Element).parentElement?.querySelector('.oxd-table-header-sort-dropdown-item');
```

Comment:
Ruchira made a change to indicate the current sorting order in the pop-up. Do we need to make any change for this based on that logic? @Super-Chama do you think this is the responsibility of this component?